script/update/validators/SetMinValidatorNodeSignatures.s.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.26;
+
+/* solhint-disable no-console*/
+import {console} from "forge-std/console.sol";
+
+import {AccessController} from "../../../src/access/AccessController.sol";
+import {PantosForwarder} from "../../../src/PantosForwarder.sol";
+
+import {PantosBaseAddresses} from "./../../helpers/PantosBaseAddresses.s.sol";
+import {SafeAddresses} from "./../../helpers/SafeAddresses.s.sol";
+
+/**
+ * @title SetMinValidatorNodeSignatures
+ *
+ * @notice Set the minimum number of required validator node signatures.
+ *
+ * @dev Usage
+ * forge script ./script/update/parameters/SetMinValidatorNodeSignatures.s.sol --rpc-url <rpc alias>
+ *      --sig "roleActions(address,address,uint256)" <newMinimumThreshold> \
+ *      <accessControllerAddress> <pantosForwarder>
+ */
+contract SetMinValidatorNodeSignatures is PantosBaseAddresses, SafeAddresses {
+    AccessController accessController;
+    PantosForwarder public pantosForwarder;
+
+    function roleActions(
+        uint256 newMinimumThreshold,
+        address accessController_,
+        address pantosForwarder_
+    ) public {
+        accessController = AccessController(accessController_);
+        pantosForwarder = PantosForwarder(pantosForwarder_);
+
+        require(
+            newMinimumThreshold > 0,
+            "Minimum threshold must be greater than 0"
+        );
+        uint256 oldThreshold = pantosForwarder
+            .getMinimumValidatorNodeSignatures();
+        require(
+            oldThreshold != newMinimumThreshold,
+            "New threshold is the same as the old one"
+        );
+
```

Comment:
I think, it will be good to check if newMinimumThreshold <= validatorNodes.length

```
address[] memory validatorNodes = pantosForwarder.getValidatorNodes();
require(
            newMinimumThreshold <= validatorNodes.length,
            "New threshold is more that number of validator nodes"
        );
```